x-pack/test/plugin_api_integration/test_suites/task_manager/task_management.ts

```diff
@@ -646,77 +646,67 @@ export default function ({ getService }: FtrProviderContext) {
       const historyItem = random(1, 100);
       const scheduledTask = await scheduleTask({
         taskType: 'sampleTask',
-        schedule: { interval: '30m' },
+        schedule: { interval: '1h' },
         params: { historyItem },
       });
 
       await retry.try(async () => {
         expect((await historyDocs()).length).to.eql(1);
-        const tasks = (await currentTasks()).docs;
+        const task = await currentTask(scheduledTask.id);
 
-        expect(getTaskById(tasks, scheduledTask.id).enabled).to.eql(true);
+        expect(task.enabled).to.eql(true);
       });
 
       // disable the task
       await bulkDisable([scheduledTask.id]);
 
       await retry.try(async () => {
-        const tasks = (await currentTasks()).docs;
-
-        expect(getTaskById(tasks, scheduledTask.id).enabled).to.eql(false);
+        const task = await currentTask(scheduledTask.id);
+        expect(task.enabled).to.eql(false);
       });
 
       // re-enable the task
       await bulkEnable([scheduledTask.id], true);
 
       await retry.try(async () => {
-        const tasks = (await currentTasks()).docs;
+        const task = await currentTask(scheduledTask.id);
 
-        expect(getTaskById(tasks, scheduledTask.id).enabled).to.eql(true);
-
-        // should get a new document even tho original schedule interval was 30m
-        expect((await historyDocs()).length).to.eql(2);
+        expect(task.enabled).to.eql(true);
+        expect(Date.parse(task.scheduledAt)).to.greaterThan(Date.parse(scheduledTask.scheduledAt));
```

Comment:
```suggestion
        expect(Date.parse(task.scheduledAt)).to.be.greaterThan(Date.parse(scheduledTask.scheduledAt));
```

I don't know if the `be` matters?